samples/core/resource_ops/resource_ops.py

```diff
@@ -13,45 +13,82 @@
 # limitations under the License.
 
 
-"""Note that this sample is just to show the ResourceOp's usage.
-
-It is not a good practice to put password as a pipeline argument, since it will
-be visible on KFP UI.
 """
+This example demonstrates how to use ResourceOp to specify the value of env var.
+"""
+
+import json
 import kfp
 import kfp.dsl as dsl
 from kubernetes import client as k8s_client
+from string import Template
+
+
+_ENV_CONFIG = """
+{
+    "apiVersion": "v1",
+    "kind": "ConfigMap",
+    "metadata": {
+        "name": "sample-env-config"
+    },
+    "data": {
+        "SPECIAL_LEVEL": "level",
+        "SPECIAL_TYPE": "type"
+    }
+}    
+"""
+
+_CONTAINER_MANIFEST = """
+{
+    "apiVersion": "batch/v1",
+    "kind": "Job",
+    "metadata": {
+        "generateName": "resourceop-basic-job-"
+    },
+    "spec": {
+        "template": {
+            "metadata": {
+                "name": "resource-basic"
+            },
+            "spec": {
+                "containers": [{
+                    "name": "sample-container",
+                    "image": "k8s.gcr.io/busybox",
+                    "command": ["/bin/sh", "-c", "env"],
+                    "envFrom": [{
+                        "configMapRef": {
+                            "name": "sample-env-config"
```

Comment:
The name should probably come from `env_config_map_op.output`.

Reply:
Talked offline, removed configmap part.